contracts/Tokens/VTokens/VTokenInterfaces.sol

```diff
@@ -114,6 +123,44 @@ contract VTokenStorage {
      * @notice Mapping of account addresses to outstanding borrow balances
      */
     mapping(address => BorrowSnapshot) internal accountBorrows;
+
+    /**
+     * @notice Underlying asset for this VToken
+     */
+    address public underlying;
+
+    /**
+     * @notice Implementation address for this contract
+     */
+    address public implementation;
+
+    /**
+     * @notice delta block after which reserves will be reduced
+     */
+    uint public reduceReservesBlockDelta;
```

Comment:
> could we use explicitly uint256 instead of uint?
I didn't make these changes in order to keep the repository consistent.

Reply:
> Also we could pack the reduceReservesBlockDelta and reduceReservesBlockNumber  in 1 storage slot. Using uint32 or uint64 we can store blocks for the next 400 years (with uint32) and around 800 years (uint64) which I think is reasonable change, saving 1 storage slot. Maybe since there are only 2 variables that store block number we can even use uint128 and pack them in 1 struct. If we read them both at the same time this will also save reading from storage whenever we use them.
WDYT?

We need to refactor or cast the pre-existing variables, like currentBlock, which is compared with reduceBlocks variables, to account for these changes. WDYT @chechu

Reply:
It would be a good optimization that would save gas during the deployment, but not too much during the execution (AFAIK). IMO, I wouldn't do it now, to reduce the changes